content/docs/beyond-the-basics/verified-domain/index.md

```diff
@@ -0,0 +1,46 @@
+---
+ title: "Verified Domain"
+ description: "Domain Verification and Why It's Important"
+ lead: "Domain Verification and Why It's Important"
+ date: 2021-05-18T14:52:25+01:00
+ lastmod: 2021-05-18T14:52:25+01:00
+ draft: false
+ images: []
+ menu:
+   docs:
+     parent: "beyond-the-basics"
+ weight: 43
+ toc: true
+---
+
+## What is domain verification?
+
+Domain verification assures that actors claiming to be part of an organization are authorized to share information on their behalf. If an organization's tenancy has been verified by the RKVST team, a badge indicating that they have been verified will appear next to their domain name.
+
+{{< note >}}
+**Note:** Having a verified domain is different from a `Tenant Display Name`. Tenant display names are internal, appearing only within your own tenancy, and are not visible to anyone you share with. A verified domain name must be set by the RKVST team, and will be visible to actors outside your tenancy. 
+{{< /note >}}
+
+## Why is it important to verify my organization's domain?
+
+Getting your organization's domain verified indicates that you are who you say you are to those you wish to share with. This helps close the trust gap inherent to information sharing between organizations or with the public.
```

Comment:
I might relax the wording around "...to those you wish to share with" since if your tenant is verified on a public asset - everyone is gonna see that.

Reply:
That's true, I went ahead and removed that part of the sentence because it reads better without.